pkg/agent/controller/traceflow/packetin_test.go

```diff
@@ -28,7 +28,24 @@ import (
 	crdv1alpha1 "antrea.io/antrea/pkg/apis/crd/v1alpha1"
 )
 
+func prepareMockTables() {
```

Comment:
Of course, but I think this could be modified in latter PR as its only used once. I think it's also ok to remove `prepareMockTables` and use `openflow.SetMockTableIDs` with passing a map directly.